generators/app/templates/app/config/route-config.js

```diff
@@ -13,7 +13,13 @@ function registerRoutes(application) {
     var route = getRoute(routeItem);
     var method = getMethod(routeItem);
 
-    registerRoute(application, controller, route, method);
+    if (method.constructor === Array) {
```

Comment:
I would use instanceof here instead of the constructor property.

if (method instanceof Array) {
  ...
}